api/src/main/java/org/openmrs/module/pihcore/identifier/sierraLeone/ConfigureSierraLeoneIdGenerators.java

```diff
@@ -31,7 +29,7 @@ public static void configurePrimaryIdentifierSource(IdentifierSourceService iss,
                     9,
                     "1234567890",
                     "100001",
-                    SierraLeoneLocations.WELLBODY_CLINIC
```

Comment:
I understand the desire not to create a constant for a single use, but then maybe a comment here saying something like "uuid for Wellbody Clinic, defined in locations.csv in config-pihsl"?